mantidproject/mantid
# Modernise namespaces to use nested specifier

**Description of work.**
Uses modern C++-17 namespaces across the project. The pros of which are:
- Simpler to read
- Less boilerplate
- Style is the same everywhere so less mental burden keeping track

This allows us to use one unified style, so I've updated the developer standards

**To test:**
- Check everything builds
- Explicitly check `ViewPort.cpp` as there was a merge conflict here


*This does not require release notes* because it is not a user visible change



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
